src/Documentation/Getting-Started-With-Orleans/Grains.md

```diff
@@ -19,13 +19,13 @@ Orleans aims to greatly simplify building a scalable application and eliminate m
 
 A typical grain encapsulates state and behavior of a single entity (e.g. a specific user or a device or a session).
 
-### Grain identity
+### Grain Identity
 
 An individual grain is a uniquely addressable instance of a grain type (class).
 Each grain has a unique identity, also referred to as a grain key, within its type.
 Grain identity within its type can be a long integer, a GUID, a string, or a combination of a long+string or GUID+string.
 
-### Accessing a grain
+### Accessing a Grain
```

Comment:
> A call to `GetGrain`  is an inexpensive local operation of constructing a grain reference with an embedded identity of the target grain. 

I think we can move this line in the deleted file to here. I think advanced users may want to know this when they are analyzing performance for their application